<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSVNCheckoutDialog</class>
 <widget class="QDialog" name="QSVNCheckoutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Checkout</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,1,0" columnstretch="1,0,0,0">
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_repository">
     <property name="title">
      <string>Repository</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
      <item row="0" column="0">
       <widget class="QLabel" name="label_URL">
        <property name="text">
         <string>URL of repository:</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_URL</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_URL">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_URL">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_dir">
        <property name="text">
         <string>Checkout directory:</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_dir</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_dir">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_multipleCopies">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Multiple, independent working copies</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="lineEdit_dir"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_depth">
     <property name="title">
      <string>Checkout depth</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,0">
      <item row="0" column="0" colspan="2">
       <widget class="QComboBox" name="comboBox_depth">
        <item>
         <property name="text">
          <string>Fully recursive</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Immediate children, including folders</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Only file children</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Only this item</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_omit">
        <property name="text">
         <string>Omit externals</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_choose">
        <property name="text">
         <string>Choose items...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_revision">
     <property name="title">
      <string>Revision</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1,0">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_head">
        <property name="text">
         <string>HEAD revision</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_revision">
        <property name="text">
         <string>Revision</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_revision"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_revision">
        <property name="text">
         <string>Show log</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_status"/>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton_ok">
     <property name="text">
      <string>&amp;OK</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="pushButton_help">
     <property name="text">
      <string>&amp;Help</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox_URL</tabstop>
  <tabstop>pushButton_URL</tabstop>
  <tabstop>lineEdit_dir</tabstop>
  <tabstop>pushButton_dir</tabstop>
  <tabstop>checkBox_multipleCopies</tabstop>
  <tabstop>comboBox_depth</tabstop>
  <tabstop>checkBox_omit</tabstop>
  <tabstop>pushButton_choose</tabstop>
  <tabstop>radioButton_head</tabstop>
  <tabstop>radioButton_revision</tabstop>
  <tabstop>lineEdit_revision</tabstop>
  <tabstop>pushButton_revision</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>pushButton_help</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>QSVNCheckoutDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>QSVNCheckoutDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
